src/config/dynamic-theme-fixes.config

```diff
@@ -6206,17 +6206,61 @@ gog.com
 
 INVERT
 i.icn.icn--close
+svg.icon-wrapper-icon
+svg.big-spot__add-to-cart-icon
+svg.ic-svg:not(.button__icon):not(.productcard-slider__nav-icon)
+.productcard-thumbnails-slider-pagination
+.carousel-pagination__page
+.big-spot__carousel-pages-container
+.big-spots__arrow-icon
+.discover-games-more__icon
 
 CSS
 .product-tile__info:hover {
     background-color: var(--darkreader-neutral-background) !important;
 }
+a.product-tile__content {
+    background-color: var(--darkreader-neutral-background) !important;
+}
+.explore-galaxy {
+    border-top: 0 !important;
+}
+.galaxy-tooltip__layer {
+    background: ${#d9d9d9} !important;
+    color: var(--darkreader-neutral-text) !important;
+}
+.big-spot__content, .big-spot__text, .big-spot__action {
+    backdrop-filter: brightness(50%) !important;
+}
+.big-spot__title, .big-spot__super-title, .big-spot__action {
+    filter: brightness(130%) !important;
+}
+.wishlist-button, .review__read-more, .review-new__add-button, .review-new__guideline-button {
+    background: none !important;
+}
 
 IGNORE IMAGE ANALYSIS
 .menu-anonymous__shelf
 
 ================================
 
+gog.com/promo
+
+CSS
+.wrapper.cf > .content.cf::before {
+    position: fixed;
+    left: 0;
+    top: 0;
+    width: 100%;
+    height: 100%;
+    background-color: rgba(0,0,0,0.5);
```

Comment:
A lot of scary code, could you link me a example page where this is used?

Reply:
![image](https://user-images.githubusercontent.com/30559591/143134090-ae8fc254-5d9c-4fc2-ab37-bbe68dad4233.png)

Sure, just click any of the three promos here to see how they look without that overlay. I tried multiple approaches, but the problem is that they use really bright background-images for container elements and after inversions you get bright text on bright background.